base_layer/core/src/base_node/comms_interface/comms_request.rs

```diff
@@ -83,6 +83,8 @@ pub enum NodeCommsRequest {
     FetchUnspentUtxosInBlock {
         block_hash: BlockHash,
     },
+    FetchOutputByPayRef([u8; 32]),
```

Comment:
```suggestion
    FetchOutputByPayRef(FixedHash),
```
we use 32byte hashes all over and parse this as fixedhash so that we don't have to handle conversions everywhere